app/src/androidTest/java/com/github/h3lp3rs/h3lp/presentation/PresArrivalActivityTest.kt

```diff
@@ -11,29 +10,21 @@ import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 
-
 @RunWith(AndroidJUnit4::class)
 class PresArrivalActivityTest {
     @get:Rule
     val testRule = ActivityScenarioRule(PresArrivalActivity::class.java)
 
+    private fun checkIsDisplayed(id: Int) {
+        onView(withId(id)).check(matches(isDisplayed()))

```

Comment:
Could also be factored out due to its several uses in test files

Reply:
Yes, like I said before, I think the time is ripe to do a testing utils class.